### ISSUE TITLE
Example page empty

### ISSUE THREAD

**Issue by madhavajay** [mpclarkson/perfect-swift-linux#3]
```
vagrant@vagrant-ubuntu-wily-64:~$ cd /vagrant && perfectserverhttp
Loaded AuthenticatorServer.so
Loaded TapTrackerServer.so
Loaded UploadEnumerator.so
/foo
    /bar
        /baz+h
    /*
        /baz+h

POST:
/raw+h
/user
    /{id}
        /baz+h

GET:
/+h
/index.html+h
/user
    /{id}
        /baz+h
    /{id}+h

Loaded URLRouting.so
Loaded WebSocketsServer.so
Starting HTTP server on 0.0.0.0:8181 with document root ./webroot/
```

When I visit: http://127.0.0.1:8181/
I see:

Index handler: You accessed path /

**Comment by mpclarkson** [comment]
Yes, that's the response the handler returns in that Example project - it basically just echo's the path.

**Comment by mpclarkson** [comment]
I originally had a "Welcome to Perfect Server" index page but removed it as I wanted their examples presented as they are.

**Comment by madhavajay** [comment]
Okay, so I built the example Tap Tracker on the simulator and got this:

Request succeeded with data Optional("The file /TapTracker was not found.")
JSON decoding failed with exception SyntaxError("Malformed boolean literal")

**Comment by madhavajay** [comment]
Authenticator Client crashed after filling the sign up form and hitting register with:

fatal error: 'try!' expression unexpectedly raised an error: Error Domain=NSCocoaErrorDomain Code=3840 "Invalid value around character 0." UserInfo={NSDebugDescription=Invalid value around character 0.}: file /Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-700.1.101.15/src/swift/stdlib/public/core/ErrorType.swift, line 50

**Comment by mpclarkson** [comment]
Try /TapTracker.mustache - that works for me.  I haven't looked into the configuration in detail. Perhaps you can drop the .mustache using Apache? Any application bugs should probably be directed to Perfect. I'm in the process of building a demo web app for my own interest and I'll share that with you - then we can debug my code rather than Perfect's demos.

**Comment by zhuhejin** [comment]
firstly i got the "Hello, world! The server is running." but a while after i got same problem as @madhavajay
